src/components/InventoryV2/InventoryViewForNursery.tsx

```diff
@@ -21,10 +26,16 @@ export default function InventoryViewForNursery(): JSX.Element {
   const [modified, setModified] = useState<number>(Date.now());
 
   const nurseryId = Number(pathParams.nurseryId);
+  const openBatchNumber = (query.get('batch') || '').toLowerCase();
 
-  // TODO in SW-4392
-  // tslint:disable-next-line:no-empty
-  const onUpdateOpenBatch = () => {};
+  const setBatchNumber = (batchNum: string | null) => {
+    if (batchNum === null) {
+      query.delete('batch');
+    } else {
+      query.set('batch', batchNum);
+    }
+    history.replace(getLocation(location.pathname, location, query.toString()));
```

Comment:
does this open up the batch modal for a specific batch?

Reply:
No this just adds it to the history. Within the table, when a batch is selected, the modal opens and calls this passed in handler, which is just responsible for history and URL modification. This was taken from `src/components/InventoryV2/InventoryView.tsx`